code/chart.js

```diff
@@ -1,4 +1,39 @@
 //DOM-selector for the canvas 👇
-const ctx = document.getElementById('chart').getContext('2d')
+const ctx = document.getElementById('myChart').getContext('2d')
 
 //"Draw" the chart here 👇
+const drawChart = (amount) =>
+{
+    const config = {
+      type: "bar",
+      data: {
+        labels: ["Completed projects", "Remaining projects"],
+        datasets: [
+          {
+            label: "Technigo projects",
+            data: [amount, 21-amount],
+            backgroundColor: [
+              "#c8cccf",
+              "#778088",
+            ],
+            barThickness: "30",
+          },
+        ],
+      },
+      options: {
+        scales: {
+          x: {
+            grid: {
+              display: false,
+            }
+          },
+          y: {
+            grid: {
+              display: false
+            }
+          },
+        }
+      }
+    }
+ const myChart = new Chart(ctx, config)
```

Comment:
This works fine, just want to point out that there is no need to assign `new Chart(ctx, config)` to a variable because you don't need to use the variable elsewhere. :)